### PR TITLE
feat: added sponsor section

### PR DESCRIPTION
**Pull request by jayeshxn**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: src/components/SponsorSection/SponsorSection.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          </a>
+        </TitleSponsorContainer>
+      ))}
+    </TitleSponsorWrapper>
+    <SponsorWrapper>
+      {sponsors.map(({ id, logo, alt, link, type, color }) => (
+        <SponsorContainer key={id}>
+          <a href={link} target='_blank noopener noreferer'>
+            <SponsorLogo src={logo} alt={alt} />
+            <SponsorText color={color}>{type}</SponsorText>
```
Comment (Shurtu-gal) [inline src/components/SponsorSection/SponsorSection.jsx:35]: Add a gap b/w them.

File: src/components/SponsorSection/SponsorSection.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  SponsorWrapper,
+  TitleSponsorContainer,
+  TitleSponsorLogo,
+  TitleSponsorWrapper,
+} from './styles';
+import Sponsors from '../../../config/content/sponsors';
+
+const { titleSponsor, sponsors } = Sponsors;
+const SponsorSection = () => (
+  <SponsorSectionContainer>
```
Comment (Shurtu-gal) [inline src/components/SponsorSection/SponsorSection.jsx]: Wrap it inside section layout see FAQ section.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/components/SponsorSection/SponsorSection.jsx (hunk 2 above)
Comment (jayeshxn) [reply src/components/SponsorSection/SponsorSection.jsx]: @Shurtu-gal I've made the changes. Please review the PR

**Review by Shurtu-gal — APPROVED** [review]
LGTM :robot:

**Pull request merged**